pkg/branding/brandedlogo_test.go

```diff
@@ -33,7 +37,8 @@ func TestGetBrandedLogo(t *testing.T) {
 	for name, tt := range tests {
 		t.Run(name, func(t *testing.T) {
 			t.Setenv("ROX_PRODUCT_BRANDING", tt.productBrandingEnv)
-			receivedLogoHash := sha256.Sum256([]byte(GetLogoBase64()))
+			logoHashBytes := sha256.Sum256([]byte(GetLogoBase64()))
+			receivedLogoHash := hex.EncodeToString(logoHashBytes[:])
```

Comment:
Suggestion: just inline this
```suggestion
			receivedLogoHash := hex.EncodeToString(sha256.Sum256([]byte(GetLogoBase64()))[:])
```

Reply:
That actually doesn't work, the compiler complains saying "Cannot slice 'sha256.Sum256([]byte(GetLogoBase64()))' (type '[32]byte')", hence I'm saving it in `logoHashBytes`